docs/photoniq/est/est-settings.md

```diff
@@ -72,3 +72,17 @@ Define how EST handles cookies.
 ### Origin Settings
 
 In the **Origin** field, enter the origin that will connect to the EST service.
+
+### Stream
+
+This option is only available if streams are enabled on your account.
```

Comment:
PhotonIQ EST customer is agnostic to GDN

Reply:
Unclear what you want me to put here.

Reply:
we need to remove this point:  "This option is only available if streams are enabled on your account..."  as EST customer is not GDN customer. Also EST customer wont be provided with GDN URL or tenant information.

Reply:
No, but the EST stream function needs to be enabled, right? Or did I misunderstand? I thought that an engineer had to toggle streams before the option was even visible to turn on or off.